### PR TITLE
Introduce state Issue #13

### PR DESCRIPTION
**Pull request by Jaff1**
you've seen the changes in action: I've stuck these on a branch so there are no clashes with th other changes - but if you review then we can merge to master. Cheers Guru.

### CODE REVIEW

File: src/Gallery-get.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import deathNote from './lightdeathnoteposter.jpg';
 import gameOfThrones from './GameOfThrones.jpg';
 import walkingDead from './The-Walking-Dead-Season-3.jpg';
 import wire from './TheWire.jpg';
+// Introduce component state #13
```
Comment (rmallols) [inline src/Gallery-get.js]: Please remove commented code :)

**Review by rmallols — CHANGES_REQUESTED** [review]
Just one subtle change requested :)

File: src/Gallery-get.js (hunk 1 above)
Comment (Jaff1) [reply src/Gallery-get.js]: changed as requested and pushed..thanks for your suggestion
.cheers

**Review by rmallols — APPROVED** [review]
Thanks buddy :)

**Comment by Jaff1** [comment]
Introduced state #13
added via a branch. merging back to master

**Pull request merged**